### PR TITLE
New API V2

### PR DESCRIPTION
**Pull request by mdebauchez**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: .github/workflows/web.yml [hunk 1]
```diff
@@ -76,19 +76,19 @@ jobs:
...
-      module_directory: centreon
-      module_name: centreon-web
-      major_version: ${{ needs.get-version.outputs.major_version }}
-      minor_version: ${{ needs.get-version.outputs.minor_version }}
-      stability: ${{ needs.get-version.outputs.stability }}
-    secrets:
-      veracode_api_id: ${{ secrets.VERACODE_API_ID_WEB }}
-      veracode_api_key: ${{ secrets.VERACODE_API_KEY_WEB_2 }}
-      veracode_srcclr_token: ${{ secrets.VERACODE_SRCCLR_TOKEN }}
+  # veracode-analysis:
```
Comment (sc979) [inline .github/workflows/web.yml:79]: Kindly, restore the pipeline

**Review by sc979 — CHANGES_REQUESTED** [review]

File: .github/workflows/web.yml (hunk 1 above)
Comment (jdelpierre) [reply .github/workflows/web.yml:79]: This is to speed up builds for testing, once ok, everything will be reactivated.

**Comment by sc979** [comment]
Thanks @mdebauchez for the modification as a draft

**Comment by jdelpierre** [comment]
Since this PR was opened a while ago and the merge bases have changed, there's a lot of conflict. A new, clearer and healthier develop base has been opened here so that @mdebauchez  can get the latest changes more easily.

**Comment by jdelpierre** [comment]
Rewrite here : https://github.com/centreon/centreon/pull/2571

**Pull request closed without merging**